cmd/controller/metrics.go

```diff
@@ -55,6 +58,25 @@ var (
 		Name:      "condition_info",
 		Help:      "Current SealedSecret condition status. Values are -1 (false), 0 (unknown or absent), 1 (true)",
 	}, []string{labelNamespace, labelName, labelCondition})
+
+	counter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: metricNamespace,
+			Name:      "api_requests_total",
```

Comment:
https://prometheus.io/docs/practices/naming/

```suggestion
			Name:      "http_requests_total",
```